scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -3542,7 +3542,7 @@ const char *pro_rad1_doc = R"(
     c : array_like
         Spheroidal parameter
     x : array_like
-        Real parameter (``|x| < 1.0``)
+        Real parameter (``x > 1.0``)
```

Comment:
There may be other places this needs to be changed; e.g. line 3534 above.

Reply:
I think I got the duplicated mistakes for the 8 function doc changes now.